scipy/optimize/tests/test_hungarian.py

```diff
@@ -41,34 +44,379 @@ def test_linear_sum_assignment():
          []),
     ]:
         cost_matrix = np.array(cost_matrix)
-        row_ind, col_ind = linear_sum_assignment(cost_matrix)
+        row_ind, col_ind = _hungarian.linear_sum_assignment(cost_matrix)
         assert_array_equal(row_ind, np.sort(row_ind))
         assert_array_equal(expected_cost, cost_matrix[row_ind, col_ind])
 
         cost_matrix = cost_matrix.T
-        row_ind, col_ind = linear_sum_assignment(cost_matrix)
+        row_ind, col_ind = _hungarian.linear_sum_assignment(cost_matrix)
         assert_array_equal(row_ind, np.sort(row_ind))
         assert_array_equal(np.sort(expected_cost),
                            np.sort(cost_matrix[row_ind, col_ind]))
 
 
 def test_linear_sum_assignment_input_validation():
-    assert_raises(ValueError, linear_sum_assignment, [1, 2, 3])
+    assert_raises(ValueError, _hungarian.linear_sum_assignment, [1, 2, 3])
 
     C = [[1, 2, 3], [4, 5, 6]]
-    assert_array_equal(linear_sum_assignment(C),
-                       linear_sum_assignment(np.asarray(C)))
-    assert_array_equal(linear_sum_assignment(C),
-                       linear_sum_assignment(np.matrix(C)))
+    assert_array_equal(_hungarian.linear_sum_assignment(C),
+                       _hungarian.linear_sum_assignment(np.asarray(C)))
+    assert_array_equal(_hungarian.linear_sum_assignment(C),
+                       _hungarian.linear_sum_assignment(np.matrix(C)))
 
     I = np.identity(3)
-    assert_array_equal(linear_sum_assignment(I.astype(np.bool)),
-                       linear_sum_assignment(I))
-    assert_raises(ValueError, linear_sum_assignment, I.astype(str))
+    assert_array_equal(_hungarian.linear_sum_assignment(I.astype(np.bool)),
+                       _hungarian.linear_sum_assignment(I))
+    assert_raises(ValueError, _hungarian.linear_sum_assignment, I.astype(str))
 
     I[0][0] = np.nan
-    assert_raises(ValueError, linear_sum_assignment, I)
+    assert_raises(ValueError, _hungarian.linear_sum_assignment, I)
 
     I = np.identity(3)
     I[1][1] = np.inf
-    assert_raises(ValueError, linear_sum_assignment, I)
+    assert_raises(ValueError, _hungarian.linear_sum_assignment, I)
+
+
+"""================= New Tests ================="""
+
+
+def test_transposing(matrix_for_tests):
+    munkres = _hungarian.Munkres(matrix_for_tests)
+    assert(munkres.marked.shape == (3, 4))
```

Comment:
style: `assert` with one space behind it, and no outer brackets around the thing being asserted